src/scss/_variables.scss

```diff
@@ -904,3 +904,11 @@ $offcanvas-border-color: var(--#{$prefix}border-color) !default;
 // Placeholder
 $placeholder-opacity-min: 0.1 !default;
 $placeholder-opacity-max: 0.2 !default;
+
+//tags
+$tag-bg-color: #ffffff !default;
```

Comment:
all variables should started on `$tag-XXX` and be groupped in Tags sections, Bg color should be inherited from `bg-surface` CSS variable